SelectBox and TagBox: add customItemCreateEvent description (#4290)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxSelectBox/1 Configuration/customItemCreateEvent.md

---
---
##### shortDescription
Specifies the <a href="https://en.wikipedia.org/wiki/DOM_events" target="_blank">DOM event</a> after which the custom item should be created. Applies only if [acceptCustomValue]({basewidgetpath}/Configuration/#acceptCustomValue) is enabled.

---

> The default custom item creating event is `change`, and items are created when users press the **Enter** key, or when the component loses focus.

Reviewer: ```suggestion
The default custom item creation event is `change`, and items are created when users press the **Enter** key or when the component loses focus.
```

To allow users to create items only when they press the **Enter** key, pass an empty string to this property.

> The following code shows how to enable item creation on **Space** key pressed down:

Reviewer: ```suggestion
The following code shows how to enable item creation when the **Space** key is pressed:
```
